Remove the dependency on the module File::Basename. 

This provides the ability to run the scripts using a local modules 
directory. While that ability is very handy with the source distribution 
and in testing environments, it is useless on a Debian package.

It is removed in a (currently still failed) attempt to remove the 
dependency on perl and depend on perl-base alone.

--- a/xpp/dahdi.cgi
+++ b/xpp/dahdi.cgi
@@ -6,8 +6,6 @@
 # modify it under the same terms as Perl itself.
 
 use strict;
-use File::Basename;
-BEGIN { my $dir = dirname($0); unshift(@INC, "$dir", "$dir/perl_modules"); }
 
 use CGI::Pretty qw/:standard start_ul start_li start_div start_pre/;
 use Dahdi;
--- a/xpp/dahdi_drivers
+++ b/xpp/dahdi_drivers
@@ -1,7 +1,5 @@
 #! /usr/bin/perl -w
 use strict;
-use File::Basename;
-BEGIN { my $dir = dirname($0); unshift(@INC, "$dir", "$dir/perl_modules"); }
 
 use Dahdi::Hardware;
 
--- a/xpp/dahdi_genconf
+++ b/xpp/dahdi_genconf
@@ -8,8 +8,6 @@
 # $Id: dahdi_genconf 5440 2008-12-05 00:24:09Z tzafrir $
 #
 use strict;
-use File::Basename;
-BEGIN { my $dir = dirname($0); unshift(@INC, "$dir", "$dir/perl_modules"); }
 
 use Dahdi;
 use Dahdi::Hardware;
--- a/xpp/dahdi_hardware
+++ b/xpp/dahdi_hardware
@@ -8,9 +8,7 @@
 # $Id: dahdi_hardware 4484 2008-06-29 16:57:48Z tzafrir $
 #
 use strict;
-use File::Basename;
 use Getopt::Std;
-BEGIN { my $dir = dirname($0); unshift(@INC, "$dir", "$dir/perl_modules"); }
 
 use Dahdi;
 use Dahdi::Span;
--- a/xpp/dahdi_registration
+++ b/xpp/dahdi_registration
@@ -8,8 +8,6 @@
 # $Id: dahdi_registration 4533 2008-07-03 08:22:02Z tzafrir $
 #
 use strict;
-use File::Basename;
-BEGIN { my $dir = dirname($0); unshift(@INC, "$dir", "$dir/perl_modules"); }
 
 use Dahdi;
 use Dahdi::Span;
--- a/xpp/lsdahdi
+++ b/xpp/lsdahdi
@@ -8,8 +8,6 @@
 # $Id: lsdahdi 4764 2008-08-12 10:21:53Z tzafrir $
 #
 use strict;
-use File::Basename;
-BEGIN { my $dir = dirname($0); unshift(@INC, "$dir", "$dir/perl_modules"); }
 
 use Dahdi;
 use Dahdi::Span;
--- a/xpp/xpp_blink
+++ b/xpp/xpp_blink
@@ -8,8 +8,6 @@
 # $Id: xpp_blink 4416 2008-06-19 17:34:36Z tzafrir $
 #
 use strict;
-use File::Basename;
-BEGIN { my $dir = dirname($0); unshift(@INC, "$dir", "$dir/perl_modules"); }
 
 use Dahdi;
 use Dahdi::Span;
--- a/xpp/xpp_sync
+++ b/xpp/xpp_sync
@@ -8,8 +8,6 @@
 # $Id: xpp_sync 4484 2008-06-29 16:57:48Z tzafrir $
 #
 use strict;
-use File::Basename;
-BEGIN { my $dir = dirname($0); unshift(@INC, "$dir", "$dir/perl_modules"); }
 
 use Dahdi::Xpp;
 use Dahdi::Xpp::Xbus;
